StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 final class PaymentSheetAnalyticsHelper {
     let analyticsClient: STPAnalyticsClient
     let isCustom: Bool
-    let configuration: PaymentSheet.Configuration
+    let configuration: IntegrationConfigurable
```

Comment:
I wonder if we should keep this `PaymentSheet.Configuration`?  I'm not sure this class can reasonably also handle analytics for EMPE

Reply:
Yeah looking at the class more, it'll be pretty hard. We'll want some embedded specific one.